podcasts/Onboarding/Models/PlusPurchaseModel.swift

```diff
@@ -56,12 +58,13 @@ class PlusPurchaseModel: PlusPricingInfoModel, OnboardingModel {
 }
 
 extension PlusPurchaseModel {
-    static func make(in parentController: UIViewController?) -> UIViewController {
+    static func make(in parentController: UIViewController?, plan: Constants.Plan, selectedPrice: UpgradeLandingView.DisplayPrice) -> UIViewController {
```

Comment:
Since the `DisplayPrice` is being used outside of just the `UpgradeLandingView` wdyt about moving it into the scope of `PlusPricingInfoModel`?

Reply:
Done in d9e2d195b7ea0d27259996faebaec2f45fdd1376